sdk/keyvault/azure-security-keyvault-administration/src/main/java/com/azure/security/keyvault/administration/implementation/KeyVaultAdministrationClientImpl.java

```diff
@@ -1805,10 +1805,10 @@ public PollerFlux<BinaryData, BinaryData> beginFullRestoreOperationAsync(BinaryD
      * @return the {@link SyncPoller} for polling of restore operation.
      */
     @ServiceMethod(returns = ReturnType.LONG_RUNNING_OPERATION)
-    public SyncPoller<BinaryData, BinaryData> beginFullRestoreOperation(BinaryData restoreBlobDetails,
+    public SyncPoller<BinaryData, BinaryData> beginPreFullRestoreOperation(BinaryData preRestoreOperationParameters,
```

Comment:
The diff on this file is just reordering methods but hard to review. We should check if we can make the generator have a consistent order of generating these methods.